core/ledger/kvledger/kv_ledger_provider.go

```diff
@@ -65,7 +65,7 @@ type Provider struct {
 	idStore              *idStore
 	blkStoreProvider     *blkstorage.BlockStoreProvider
 	pvtdataStoreProvider *pvtdatastorage.Provider
-	vdbProvider          privacyenabledstate.DBProvider
+	dbProvider           *privacyenabledstate.DBProvider
```

Comment:
Given that we have more than one stores, I feel that the earlier name `vdbProvider` was clearer. If you want to change, then `statedbProvider` may be a better choice in order to retain the context.